### PR TITLE
CORE-11412 Added virtual node status topic

### PR DESCRIPTION
**Pull request by owenstanford**
Adds support for async virtual node creation status tracking

`corda-runtime-os` PR: https://github.com/corda/corda-runtime-os/pull/3423

### CODE REVIEW

File: data/topic-schema/src/main/resources/net/corda/schema/VirtualNode.yaml [hunk 1]
```diff
@@ -82,4 +82,17 @@ topics:
...
       - rest
-    config:
\ No newline at end of file
+    config:
+  VirtualNodeOperationStatusTopic:
+    name: virtual.node.operation.status
+    consumers:
+      - db
+    producers:
+      - rest
```
Comment (davidcurrie) [inline data/topic-schema/src/main/resources/net/corda/schema/VirtualNode.yaml]: Are these not the wrong way round?

Comment (driessamyn) [reply data/topic-schema/src/main/resources/net/corda/schema/VirtualNode.yaml]: I think producer needs to be both `rest` and `db` and consumer `rest`

Comment (owenstanford) [reply data/topic-schema/src/main/resources/net/corda/schema/VirtualNode.yaml]: fixed

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Pull request merged**